examples/data_loss_prevention/README.md

```diff
@@ -122,7 +122,7 @@ Prior to running the DLP pipeline, additional requirements must be installed in
 ```bash
 conda env update --solver=libmamba \
   -n ${CONDA_DEFAULT_ENV} \
-  --file ./conda/environments/examples_cuda-128_arch-$(arch).yaml
+  --file ./conda/environments/all_cuda-128_arch-$(arch).yaml
```

Comment:
Is there a specific reason why all must be used?

Reply:
Yes, because the example needs to be compiled, and the build dependencies don't exist in the examples environment.